ci-operator/step-registry/sdn/conf/multi-manifest/sdn-conf-multi-manifest-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: sdn-conf-multi-manifest
```

Comment:
I don't know if this is a big deal but it might be cleaer to change this to sdn-conf-multitenant-manifest? it might be clearer if we ever need to create many manifests in the future

Reply:
I was keeping the length of the filename down.